test/typecheck.test.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+var check = require('../lib/typecheck');
+var isNumber = require('lodash/isNumber');
+
+describe('check', function() {
+    var errorMessage = 'It did not check correctly';
+
+    it('checks the if it passes the function the check', function() {
```

Comment:
'checks if it passes the function to check'

Reply:
Fixed!